### PR TITLE
[Sprint: 42] Xd 2684 - Update to java 1.7

### PR DESCRIPTION
**Pull request by dturanski**
Update to jdk 1.7

### CODE REVIEW

File: spring-xd-starters/spring-xd-module-parent/publish-maven.gradle [hunk 1]
```diff
@@ -162,7 +163,7 @@ def customizePom(pom, gradleProject) {
 				dependency {
 					groupId 'org.apache.hadoop'
 					artifactId 'hadoop-common'
-					version '2.3.0'
+					version '2.6.0'
```
Comment (mbogoevici) [inline spring-xd-starters/spring-xd-module-parent/publish-maven.gradle]: Ok, got it (scrap my earlier comment). Both the boot dependency and this were out of sync with `dependencies.properties`.

Comment (ericbottard) [inline spring-xd-starters/spring-xd-module-parent/publish-maven.gradle]: so, shouldn't we be using the value from the .properties file then?

Comment (mbogoevici) [inline spring-xd-starters/spring-xd-module-parent/publish-maven.gradle]: We probably could, but I would suggest creating a separate issue for it, as it's already stretching the scope of the original PR.

Comment (mbogoevici) [inline spring-xd-starters/spring-xd-module-parent/publish-maven.gradle]: LGTM; captured @ericbottard 's suggestion in https://jira.spring.io/browse/XD-2694

**Comment by mbogoevici** [comment]
Merged w/https://github.com/spring-projects/spring-xd/commit/a26badf5e1c4e1db21f2a72ef54159cff72fc055

**Pull request closed without merging**